content/en/blog/2022/otel-unplugged-kubecon-na-22

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Maybe because this post is under `/blog/2022` you can drop the `-22` file suffix?

Oh, it needs to have a `.md` extension. That's probably why `hugo` isn't picking this up.